eco-core/core-plugin/src/main/kotlin/com/willfp/eco/internal/spigot/integrations/antigrief/AntigriefLands.kt

```diff
@@ -12,28 +12,28 @@ import org.bukkit.entity.Monster
 import org.bukkit.entity.Player
 
 class AntigriefLands(private val plugin: EcoPlugin) : AntigriefIntegration {
-    private val landsIntegration = LandsIntegration(this.plugin)
+    private val landsIntegration = LandsIntegration.of(this.plugin);
```

Comment:
This doesn't need a semi-colon, eco has pretty strict code style :) It can be found/used using a checkstyle plugin or reading the readme

Reply:
Sorry I fixed the error in this commit https://github.com/Auxilor/eco/pull/228/commits/297bb10b85497fc256b40ec7f8b86c5ff62ea291 . It's a java reflex

Reply:
No you're good, I'm the same way sometimes